server/sockets.ts

```diff
@@ -244,7 +242,7 @@ export class ServerStream extends Streams.ObjectReadWriteStream<string> {
 
 					server.serve(req, res, e => {
 						if (e && (e as any).status === 404) {
-							staticServer.serveFile('404.html', 404, {}, req, res);
+							staticServer.serveFile('/404.html', res, {code: 404, thisDir: true});
```

Comment:
:|

You told me `404.html` was the default, but it's clearly explicitly specified here!

Reply:
There's code in the module to default to it - i just left this in as a fallback.

Reply:
I mean, I don't think there's code in `node-static` to default to it, which was your original reasoning for adding code to your own static file server to default to it.